sources/qa-lang-main.php

```diff
@@ -145,10 +145,10 @@
        'selected'=>'izbrano', 
        'send_button'=>'Pošlji', 
        'since_x'=>'od ^', //since ^
-       'suggest_ask'=>'Pomagaj nam začeti in ^1asking question^2.', //Help get things started by ^1asking a question^2.
-       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions v tej category^2.', //To see more, click for all the ^1questions in this category^2.
-       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2.', //To see more, click for the ^1full list of questions^2.
-       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
+       'suggest_ask'=>'Pomagaj nam začeti z ^1asking a question^2.', //Help get things started by ^1asking a question^2.
+       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions in this category^2.', //To see more, click for all the ^1questions in this category^2.
+       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full list of questions^2.', //To see more, click for the ^1full list of questions^2.
+       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full list of questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
```

Comment:
da to me je zmotilo, zato sem imel tudi toliko odprtih vej, saj sem vedno
ko sem opazil da v vašem repozitoriju ni sprememb, šel ponovno popravljati
napake, misleč da sem pozabil shraniti.

2015-04-13 20:55 GMT+02:00 Matjaž M. <notifications@github.com>:

> In sources/qa-lang-main.php
> <https://github.com/matmr/q2a-prevajanje-FS/pull/9#discussion_r28267602>:
>
> > @@ -145,10 +145,10 @@
> >         'selected'=>'izbrano',
> >         'send_button'=>'Pošlji',
> >         'since_x'=>'od ^', //since ^
> > -       'suggest_ask'=>'Pomagaj nam začeti in ^1asking question^2.', //Help get things started by ^1asking a question^2.
> > -       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions v tej category^2.', //To see more, click for all the ^1questions in this category^2.
> > -       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2.', //To see more, click for the ^1full list of questions^2.
> > -       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full seznam questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
> > +       'suggest_ask'=>'Pomagaj nam začeti z ^1asking a question^2.', //Help get things started by ^1asking a question^2.
> > +       'suggest_category_qs'=>'Če si želite ogledati več, kliknite za vse ^1questions in this category^2.', //To see more, click for all the ^1questions in this category^2.
> > +       'suggest_qs'=>'Če si želite ogledati več kliknite za ^1full list of questions^2.', //To see more, click for the ^1full list of questions^2.
> > +       'suggest_qs_tags'=>'Če si želite ogledati več kliknite za ^1full list of questions^2 ali ^3popular tags^ 4.', //To see more, click for the ^1full list of questions^2 or ^3popular tags^4.
>
> Po kratkem razmisleku -- vas morda moti da spremembe še niso vidne v mojem
> repozitoriju? Ker to je normalno in ne bodo dokler ne odobrim pull requesta.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matmr/q2a-prevajanje-FS/pull/9/files#r28267602>.
>